pkg/backporter/backporter_test.go

```diff
@@ -173,7 +145,17 @@ func TestGetClonesHandler(t *testing.T) {
 			},
 			ResCheck{
```

Comment:
Nit: Tests are easier to write and to grok if they don't have nested attributes so in this case make everything that is part of `ResCheck` an attribute of the test case and then have three `if actualThing != tc.expectedThing { t.Errorf("actual thing does not match expected thing")}`. Golang will not abort the test if you use `Errorf` rather than `Fatalf` so the user will still see all issues in case multiple checks fail.

Also something like this type should not be shared among tests, as it results in the type being a superset of what all tests that use it need.

Reply:
Alright I have moved all of them into the outer struct and gotten rid of the ResCheck class